packages/semantic-release-commit-rules/lib/commit-analyzer.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @fileoverview Functionality for the `@semantic-release/commit-analyzer` plugin
+ */
+
+/**
+ * Converts a commit message configuration object in to a Semantic Release
```

Comment:
```suggestion
 * Converts a commit message configuration object into a semantic release
```
(isn't capitol anywhere else)